frontend/automation/cypress/support/cleanup.js

```diff
@@ -4,4 +4,32 @@ before(() => {
 
 after(() => {
   cy.clean()
+
+  // Stop any running services
+  cy.window().then(win => {
+    const handler = win.go.backend.Handler
+
+    return handler
+      .GetServiceState()
+      .then(res => {
+        if (res.running) {
+          return handler.StopService()
+        }
+        return Promise.resolve()
+      })
+      .then(() => handler.GetConsoleState())
+      .then(res => {
+        if (res.running) {
+          return handler.StopConsole()
+        }
+        return Promise.resolve()
+      })
+      .then(() => handler.GetTokenDAppState())
+      .then(res => {
+        if (res.running) {
+          return handler.StopTokenDApp()
+        }
+        return Promise.resolve()
+      })
```

Comment:
chore blocking): can we use async await and call `done` from mocha? May fix it hanging, as right now this is a race condition as the hook doesn't know it's async

Reply:
We can make the .then callback async (which I've done) but we can't make the hook itself async as cy commands shouldn't be awaited, at least not without using something like [cypress-promise](https://www.npmjs.com/package/cypress-promise)